### DISCUSSION TITLE
@types/[matter-js]

### DISCUSSION THREAD

**Discussion by jonlepage** [DefinitelyTyped/DefinitelyTyped#63739] [Issues with a @types package] [answered]
we can unasign contraint relation by set null.
`node_modules/@types/matter-js/index.d.ts`
![image](https://user-images.githubusercontent.com/24865815/209589445-97cacb17-e8b0-42bd-9121-e0df5c5b8c83.png)

![image](https://user-images.githubusercontent.com/24865815/209589488-0f995bcb-8e4e-475f-8f9d-309b39e3ed0e.png)

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
please open PR

**Reply by jonlepage** [reply]
idht sry, do it :)

**Comment by phunanon** [comment]
I'm having the same problem - I'll look into how difficult creating a PR is, and do it!

**Reply by phunanon** [reply]
I've given it a go! https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65378

**Reply by stevensnoeijen** [reply]
Thanks @phunanon :) Merged and should be available soon on https://www.npmjs.com/package/@types/matter-js/v/0.18.3 :)

**Reply by phunanon** [reply, marked as answer]
@djmisterjon, I think you can set this post as answered now :)

**Reply by jonlepage** [reply]
thank, btw i stopped use this libs, watch your code, big issue in math who let your entities bypass wall sometime!

**Reply by stevensnoeijen** [reply]
@djmisterjon the code of matter-js is not located in this repository, this is just the typings.
Feel free to create a ticket https://github.com/liabru/matter-js to report your found issue.

**Reply by jonlepage** [reply]
> @djmisterjon the code of matter-js is not located in this repository, this is just the typings. Feel free to create a ticket https://github.com/liabru/matter-js to report your found issue.

y i know, just warn for no wasting time, it was a 3 yr old issue, the maintainer wont fix it.
i lost 5 day on this libs for nothing:)
